nn/conformer.py

```diff
@@ -44,18 +43,17 @@ class _ConformerConvBlock(nn.Module):
       FF -> GLU -> depthwise conv -> BN -> Swish -> FF
   """
 
-  def __init__(self, dim_model: int, kernel_size: int, l2: float = 0.0, batch_norm_eps: float = 1e-5,
+  def __init__(self, dim_model: int, kernel_size: int, batch_norm_eps: float = 1e-5,
       batch_norm_momentum: float = 0.1, batch_norm_other_opts=None):
     """
     :param dim_model:
     :param kernel_size:
-    :param l2:
     """
     super().__init__()
 
-    self.positionwise_conv1 = nn.Linear(n_out=dim_model * 2, l2=l2)
-    self.depthwise_conv = nn.Conv(n_out=dim_model, filter_size=(kernel_size,), groups=dim_model, l2=l2, padding='same')
-    self.positionwise_conv2 = nn.Linear(n_out=dim_model, l2=l2)
+    self.positionwise_conv1 = nn.Linear(n_out=dim_model * 2)
```

Comment:
Note, once we have #17, this doesn't really work like this.

Reply:
so better to implement GLU in the way you suggested before by splitting this into 2 modules?

Reply:
Not sure. Maybe easier for now. But we probably need to make something like this work in any case at some point. Also, I wonder which variant is actually more efficient...

Reply:
I think the variant with `dim_model * 2` is simpler though (the code is simpler). So maybe anyway use that. And we just need to make sure that the dim tag logic works here.

Reply:
Yes I agree.